TypeCobol/Compiler/Scanner/GroupToken.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TypeCobol.Compiler.Scanner
+{
+    /// <summary>
+    /// A class that represents a Group Of Tokens.
+    /// </summary>
+    public class GroupToken : Token
```

Comment:
`TokenGroup`  class already exists. Can we use it instead (with adaptations if need be) ?

Reply:
Yep... yes you are right I must test it.